scio-bigquery/src/main/scala/com/spotify/scio/bigquery/client/BigQueryConfig.scala

```diff
@@ -46,7 +48,11 @@ object BigQueryConfig {
 
   def location: String = DefaultLocation
 
-  def scopes: Seq[String] = DefaultScopes
+  def scopes: Seq[String] =
+    DefaultScopes ++
+      BigQuerySysProps.DriveScope.valueOption
+        .map(x => Try(x.toBoolean).getOrElse(false))
```

Comment:
use `filter(...).map(...)` instead?

Reply:
done. lmk if that's what you were thinking